### PR TITLE
man: add example for choosing page section

### PR DESCRIPTION
**Pull request by fetlang**
I feel like this is a useful enough option to include in the tldr page

### CODE REVIEW

File: pages/common/man.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 `man {{command}}`
 
+- Display man page for a command from section 7:
+
+`man {{command}}.7`
```
Comment (sbrl) [inline pages/common/man.md]: Perhaps we could put `7` in `{{` and `}}`?

Comment (fetlang) [reply pages/common/man.md]: oh yeh that makes sense

Comment (agnivade) [reply pages/common/man.md]: In several examples, I see `man command <section>` being used. Is that incorrect ?

Comment (fetlang) [reply pages/common/man.md]: On Linux, `man command <section>` will pull up two man pages: one for command, and one for section(which will undoubtedly not exist).

Comment (agnivade) [reply pages/common/man.md]: Ah thanks !

**Comment by agnivade** [comment]
Moved . out of tokens. Thanks.

**Pull request merged**